client/src/components/Header.jsx

```diff
@@ -53,15 +81,39 @@ const Header = () => {
           </Typography>
           <Typography color="textPrimary">Notifications</Typography>
         </Box>
+
         <Box
           display="flex"
-          width="100px"
-          px={5}
+          // width="100px"
           alignItems="center"
           justifyContent="space-between"
+          // style= {{background:"red"}}
         >
-          <Avatar />
-          <Typography color="textPrimary">Profile</Typography>
+          <Avatar className={classes.avatar} />
+          <Button
+            color="primary"
+            onClick={handleClick}
+            className={classes.profileBtn}
+          >
+            Profile
+          </Button>
+          <Popover
+            open={open}
+            anchorEl={anchorEl}
+            onClose={handleClose}
+            anchorOrigin={{
+              vertical: "bottom",
+              horizontal: "right",
+            }}
+            transformOrigin={{
+              vertical: "top",
+              horizontal: "right",
+            }}
+          >
+              <MenuItem>Profile</MenuItem>
+              <MenuItem>My account</MenuItem>
+              <MenuItem onClick= {signoutHandler} >Logout</MenuItem>
```

Comment:
Delete the space after the `=`: `onClick= {singoutHandler}` to `onClick={signoutHandler}` 🙂